doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -0,0 +1,512 @@
+:orphan:
+
+.. _ug_nrf54h20_ironside_se:
+
+IronSide Secure Element
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The IronSide Secure Element (IronSide SE) is a firmware for the Secure Domain of the nRF54H20 SoC that provides security features.
+It is provided as a precompiled binary, part of the nRF54H20 SoC bundle.
+
+IronSide SE provides the following features:
+
+* Global memory configuration (via UICR.MPCCONF)
+* Peripheral configuration (via UICR.PERIPHCONF)
+* Boot commands (ERASEALL, DEBUGWAIT)
+* CPUCONF service
+* Update service
+
+.. _ug_nrf54h20_ironside_se_uicr:
+
+Global Resource configuration
+*****************************
+
+You can configure some global resources through the applications running on local domains (like the Application core).
+Specifically, you can configure all the memory not reserved by IronSide SE, referred to as *application-owned memory*, and certain peripherals located in the global domain.
+
+This configuration is stored in the User Information Configuration Registers (``UICR``), a non-volatile memory region used to hold device-specific configuration data.
+Specifically, the configuration is located in the registers of the first Non-Volatile Register (``NVR``) page of the ``UICR``, known as ``NVR0``, as well as in the data referenced by these registers.
+The acronym ``UICR`` collectively refers to this region and its associated registers.
+
+If no configuration is defined in the ``UICR``, IronSide SE applies default configurations to configurable global resources.
+Applications that do not require changes to these defaults can run without modifying the ``UICR``.
+
+The following fields are defined in the ``UICR``:
+
+* ``VERSION`` - Specifies the version of the ``UICR``.
+* ``LOCK`` - Locks the contents of the ``UICR``.
+* ``APPROTECT`` - Specifies the protection settings for the various Access Ports (APs) in the system.
```

Comment:
```suggestion
* ``APPROTECT`` - Specifies the protection settings for the various access ports (APs) in the system.
```
No capitalization in https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/security/ap_protect.html
Global edit - please apply below as well.